kukur/source/sql.py

```diff
@@ -145,9 +145,12 @@ def get_metadata(self, selector: SeriesSelector) -> Metadata:
                     continue
                 if isinstance(value, str) and value == "":
                     continue
-                metadata.coerce_field(
-                    name, self._metadata_value_mapper.from_source(name, value)
-                )
+                try:
+                    metadata.coerce_field(
+                        name, self._metadata_value_mapper.from_source(name, value)
+                    )
+                except ValueError:
```

Comment:
The main question for me was: why is the `ValueError` thrown?

The two lines before this should already check if it's an empty string.

Reply:
The `ValueError` is not thrown when it is an empty string, this is indeed checked. But when there is a real string value that cannot be parsed as a float, the value error is thrown.

Reply:
This exact message is thrown when a string is not empty but holds one or more spaces.